Initial-Draft.md

```diff
@@ -0,0 +1,181 @@
+# Cypress Best Practices
+
+This is a best practices guide for writing integration tests with Cypress at Affirm. It draws from the recommendations set [here](https://docs.cypress.io/guides/references/best-practices/) in the Cypress official best practice documentation and include some Affirm-specific examples. 
+
+## File structure and naming conventions
+
+### Anti patterns for test organizations and naming convention
+- Naming folders after team names is not idea since those team names are garaunteed to stay the same. 
+
+Example: https://github.com/Affirm/web-ux/tree/develop/integration-tests-v2/cypress/specs/user-portal/us affirm-anywhere-adoption is no longer a team name.
+
+### Best Practice 
+- Name folders after artifacts (ex: https://github.com/Affirm/web-ux/tree/develop/integration-tests-v2/cypress/specs/user-portal/us/settings)
+
+This makes it easier to determine ownership of artifacts and directories no matter what team changes may happen to the team that owns it now.  
+
+## Cypress Commands 
+
+Cypress commands can be queries, actions, or assertions.
+
+In general, the structure of your test should flow query -> query -> command or assertion(s). **It's best practice not to chain anything after an action command.**
+
```

Comment:
Not relevant here but I was just wondering, are we encouraging / discouraging custom commands? 

https://docs.cypress.io/api/cypress-api/custom-commands

Reply:
Great question!  Can I ask you re-comment this question on the final draft PR that I plan to share with the front end Platform team? It would be great to include them in this discussion. (this rough draft PR has only been shared within our team for now.

Reply:
Sounds good. I will definitely re-comment on the final draft PR.